tests/integration/managed_array_tests.cpp

```diff
@@ -1431,18 +1491,18 @@ GPU_TEST(ManagedArray, DeviceDeepCopy)
 
   array.free();
   copy.free();
+  assert_empty_map(true);
 }
-#endif
-#endif  // defined(CHAI_ENABLE_CUDA) || defined(CHAI_ENABLE_HIP)
-
-#ifdef CHAI_ENABLE_CUDA
 
 GPU_TEST(ManagedArray, CopyConstruct)
 {
   const int expectedValue = rand();
 
   chai::ManagedArray<int> array(1, chai::CPU);
-  array[0] = expectedValue;
```

Comment:
Did this not work?

Reply:
it worked, but it was touching uninitialized memory (I wanted to clean up memcheck for these tests)

Reply:
What in particular was uninitialized?

Reply:
the contents of array